### PR TITLE
Added atlus fbn and hbn (Sierra)

### PR DESCRIPTION
**Pull request by Tupelov**

### CODE REVIEW

File: templates/atlus_fbn.bt [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	float Unknown7;
+	float Unknown8;
+	float Unknown9;
+	float Unknown10;
+	float Unknown11;
+	float Unknown12;
+	float Unknown13;
+} TFBNEntries2 <name = "FBNEntries">;
+
+typedef struct TableSegment
```
Comment (TGEnigma) [inline templates/atlus_fbn.bt:130]: This is not a good name for this struct as it's not represenative of its contents.

File: templates/atlus_hbn.bt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	u8 Unknown1;
+	u8 Unknown2;
+	u8 Unknown3;
+	float Unknown4 <name = "X Pos">;
+	float Unknown5 <name = "Z Pos">;
+	float Unknown6 <name = "Y Pos">;
+	float rot <name = "Rotation">;
+} THBNEntries2 <name = "Type 2 Entries">;
+
+typedef struct TableSegment
```
Comment (TGEnigma) [inline templates/atlus_hbn.bt:37]: This is not a good name for this struct as it's not represenative of its contents.

File: templates/atlus_hbn.bt [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+{
+    u32 Field01;
+	u16 Version_Major <name = "Version Major Number">;
+	u16 Version_Minor<name = "Version Minor Number">;
+	u32 Num_Of_Type_1_Entries <name = "Number of Type 1 Entries">;
```
Comment (TGEnigma) [inline templates/atlus_hbn.bt:42]: The name comments don't match the field name

**Review by TGEnigma — CHANGES_REQUESTED** [review]
Thanks for your contribution, however there are some issues:
* Name tags are overall very inconsistent. Either name all of them or none at all, and ensure it is similar to the field name. I personally don't think they add anything of value in most cases.
* The naming convention is inconsistent. Use PascalCase, so no underscores and no names starting with a lowercase character.

**Comment by TGEnigma** [comment]
Any updates?

File: templates/atlus_fbn.bt (hunk 1 above)
Comment (TGEnigma) [reply templates/atlus_fbn.bt:130]: Name should be FbnFile or something equivalent that represents the contents better

File: templates/atlus_hbn.bt (hunk 2 above)
Comment (TGEnigma) [reply templates/atlus_hbn.bt:37]: Name should be FbnFile or something equivalent that represents the contents better

File: templates/atlus_Nbin.bt [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+//   Purpose: 
+//  Category: 
+// File Mask: 
+//  ID Bytes: 
+//   History: 
+//------------------------------------------------
+
+struct file{
+    uint entries;
+    FSkip(28);
```
Comment (TGEnigma) [inline templates/atlus_Nbin.bt:16]: Are the skipped sections always 0?

File: templates/atlus_Nbin.bt [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+           uint32 uknown1;
+           uint32 ScriptID;
+           FSkip(16);
+    }SEC2[entries];//No idea how long this is suppossed to be only a place holder value
+            
+        
+        
+   
+}File;
+FSeek(0);
```
Comment (TGEnigma) [inline templates/atlus_Nbin.bt:40]: The code at the end of the file doesn't make much sense to me. Why is this not part of the File struct?

File: templates/atlus_fbn.bt [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+	float ShadowRadius <name = "Shadow Radius">;
+	u8 Unknown4;
+	u8 Unknown5;
+	u8 Unknown6;
+	u8 Unknown7;
+	u8 Unknown8;
+	u8 Unknown9;
+	u8 Unknown10;
+	u8 Unknown11;
+	if (Type1EntrySize==320)
```
Comment (TGEnigma) [inline templates/atlus_fbn.bt:55]: This variable doesn't exist in this scope, you should pass it to the struct constructor if you wish to use it

**Review by TGEnigma — CHANGES_REQUESTED** [review]

File: templates/atlus_Nbin.bt (hunk 4 above)
Comment (Tupelov) [reply templates/atlus_Nbin.bt:16]: These are always FFFFF

**Comment by Tupelov** [comment]
Ok hopefully now

**Pull request merged**